android/src/main/java/com/swmansion/rnscreens/ScreenStack.java

```diff
@@ -169,64 +169,64 @@ protected void performUpdate() {
       }
     }
 
-    boolean customAnimation = false;
+    // variables used for "open/close animation" logic
```

Comment:
Is this comment necessary?

Reply:
Removed in 9722871393482a56e8f10e2a77dce2679fe8c23c